extension/chrome/settings/modules/contacts.ts

```diff
@@ -114,21 +112,62 @@ View.run(class ContactsView extends View {
     Browser.saveToDownloads(exportFile);
   }
 
+  private actionRenderListPublicKeyHandler = async (viewPubkeyButton: HTMLElement) => {
+    const parentTr = $(viewPubkeyButton).closest('tr');
+    const email = parentTr.attr('email')!;
+    const contact = await ContactStore.getOneWithAllPubkeys(undefined, email);
+    const e = Xss.escape(email);
+    if (contact && contact.sortedPubkeys.length) {
+      let tableContents = '';
+      for (const pubkey of contact.sortedPubkeys) {
+        const keyid = Xss.escape(pubkey.pubkey.id);
+        const type = Xss.escape(pubkey.pubkey.type);
+        const show = `<a href="#" title="Show" class="action_show" data-test="action-show-pubkey-${keyid}-${type}"></a>`;
+        const remove = `<a href="#" title="Remove" class="action_remove" data-test="action-remove-pubkey-${keyid}"></a>`;
+        let status: string;
+        if (pubkey.revoked) {
+          status = 'revoked';
+        } else if (pubkey.pubkey?.usableForEncryption) {
+          status = 'active';
+        } else if (pubkey.pubkey?.usableForEncryptionButExpired) {
```

Comment:
why do we have `usableForEncryption`=true and `usableForSigning`=true for an expired X.509 certificate?
```
Type: x509
Fingerprint: 16BB 4074 03A3 ADC5 5E1E 0E4A F93E EC8F B187 C923
Users: actalis@meta.33mail.com
...
Expired: yes
Usable for encryption: true
Usable for signing: true
```
Is this intentional? Then it is calculated as "active"
![image](https://user-images.githubusercontent.com/23335944/119652137-68259a80-be2e-11eb-8b6e-316dc62a7c0a.png)

Reply:
> why do we have `usableForEncryption`=true and `usableForSigning`=true for an expired X.509 certificate?
> 

@tomholub Can you clarify on this,please?

Reply:
that would be a bug i think - or maybe incomplete implementation.